validator/webui/index.html

```diff
@@ -474,7 +474,22 @@
             }
           });
 
-          var minimumValidAmp =
+          // Interprets the parameters after the '#' in the URL.
+          var paramStrings = window.location.hash.substr(1).split('&');
+          var params = {};
+          for (var ii = 0; ii < paramStrings.length; ii++) {
+            var keyValue = paramStrings[ii].split('=');
+            if (keyValue[0].length > 0) {
+              params[keyValue[0]] = keyValue[1];
```

Comment:
We can't be sure there's a value in keyValue[1], but maybe you're okay with that just being undefined.

e.g. (webui.../#asdf)